PFOE-43 Komponent do tworzenia nowego wydarzenia, zakomentowałem wywo…

HubertMos99 opened the pull request:
…łanie funkcji createEvent(), wyswietlam zmienne w console.log() bo nie wiem czy jest poprawny format

HubertMos99 committed:
PFOE-43 Komponent do tworzenia nowego wydarzenia, zakomentowałem wywołanie funkcji createEvent(), wyswietlam zmienne w console.log() bo nie wiem czy jest poprawny format

MCMisha commented on frontend/pfoe-front/src/app/new-event/new-event.component.html:
```diff
@@ -1 +1,71 @@
+        </mat-error>
+        <mat-form-field class="shape-container mt-3" appearance="outline">
+          <mat-label>Data</mat-label>
+          <input matInput type="text"  onfocus="(this.type = 'date')" onfocusout="(this.type = 'text')" formControlName="eventDate"/> <!---------------------------------------------------------------------->
```
https://material.angular.io/components/datepicker/overview
Użyj datepicker

MCMisha commented on frontend/pfoe-front/src/app/new-event/new-event.component.html:
```diff
@@ -1 +1,71 @@
+        <mat-form-field class="shape-container mt-3" appearance="outline">
+          <mat-label>Kategoria</mat-label>
+          <mat-select matInput formControlName="eventCategory">
+            <mat-option value="1">Biznes</mat-option>
```
Zrób enum do tych wartości

MCMisha commented on frontend/pfoe-front/src/app/new-event/new-event.component.ts:
```diff
@@ -1,10 +1,76 @@
+      participantNumber: Number(this.newEventForm.value.eventMaxParticipants),
+      location: this.newEventForm.value.eventLocation,
+      visits_number: 0,
+      creation_date: new Date()
```
Gdzie jest pole 'organizer'?

HubertMos99 committed:
PFOE-43 Komponent do tworzenia nowego wydarzenia.
Dodane pole organizer w nowym Evencie, stworzony i użyty w formularzu enum event-category, jak używam datepickera to cały formularz znika i pokazuje błąd

MCMisha committed:
PFOE-43 Dodanie modułów do datepicker'a

MCMisha commented on frontend/pfoe-front/src/app/new-event/new-event.component.html:
```diff
@@ -1 +1,71 @@
+          Ilość miejsc jest wymagana.
+        </mat-error>
+        <mat-form-field class="shape-container mt-3" appearance="outline">
+          <mat-label>Lokacja</mat-label>
```
Lokalizacja

HubertMos99 committed:
PFOE-43 Komponent do tworzenia nowego wydarzenia.
Nie wiem czy w enumie kategoria maja być z małej czy dużej litery

MCMisha commented on frontend/pfoe-front/src/app/new-event/new-event.component.html:
```diff
@@ -1 +1,72 @@
+        </mat-error>
+        <mat-form-field class="shape-container mt-3" appearance="outline">
+          <mat-label>Data</mat-label>
+          <input matInput  [matDatepicker]="releasedAtPicker" formControlName="eventDate" [min]="todaysDate" (click)="releasedAtPicker.open()"/> <!---------------------------------------------------------------------->
```
dodaj 'readonly' obok 'matInput', aby tylko wybierać datę i ja to zmerguję

HubertMos99 committed:
PFOE-43 Komponent do tworzenia nowego wydarzenia.
dodane readonly
